src/stores/connectors/keys/import.go

```diff
@@ -0,0 +1,24 @@
+package keys
+
+import (
+	"context"
+	"github.com/consensys/quorum-key-manager/src/stores/store/entities"
+)
+
+func (c Connector) Import(ctx context.Context, id string, privKey []byte, alg *entities.Algorithm, attr *entities.Attributes) (*entities.Key, error) {
+	logger := c.logger.With("id", id).With("algorithm", alg.Type).With("curve", alg.EllipticCurve)
+	logger.Debug("importing key")
+
+	key, err := c.store.Import(ctx, id, privKey, alg, attr)
```

Comment:
how would you implement this otherwise? We need a mechanism to encrypt and decrypt private keys when you want to import them? Every wallet out there (Ledger, Metamask, etc.) allows you to import a "Seed", which is your private key, the difference is that these wallets are local

Reply:
letting this ticket to a different topic